content/concepts/kata/beta-process.md

```diff
@@ -5,98 +5,46 @@ sidebar: explanation-kata
+
+## Steps of Beta Process
+
+Some of the steps are available only after earnig the required privilege, but every user can help reviewing a kata in some way.
+
+### Attempting the Kata
+
+Training on a kata and solving it is the first step to verify its quality, but also a very effective one. Reading through description, users can check if it's clear enough, or something needs to be reworded or explained in a clearer way. When tests are run, reviewers can verify if they are sufficient, or whether assertion messages are clear and easy to understand. Users can check whether their experience with the kata meets all [Kata authoring best practices](/recipes/create-kata/best-practices/). When any issues are found, users can raise issues or sugest changes in kata discourse.
+
+While solving a beta kata, users are encouraged to go beyond just finding and submitting a working solution. It's very helpful when people solving the kata do some additional effort and test the tests themselves a bit: verify if test suite handles incorrect answers gracefully (for example, if it does not crash on `null`s), if assertion messages are clear enough, etc.
+
+When kata is successfully solved, user unlocks further steps of the review process.
+
+### Satisfaction vote
+
+Users with [sufficient privileges](/references/gamification/privileges/#privileges) are able to vote on kata quality. They can cast one of three possible votes:
+
+- **Satisfied**
+- **Somewhat satisfied**
+- **Not satisfied**
+
+If you think the kata itself is valuable but you simply have some issues with it, then you should select "Somewhat" and create an issue instead. If you thought the kata is really great, other than the issues, then you can choose "Very". Please do not choose "None" simply because the kata has an issue but it is otherwise a good kata. The satisfaction rating is meant to indicate quality of kata as a concept, not its current state of condition in terms of open issues which can be fixed.
+
+### Difficulty rating vote
+
+Kata author assigns initial difficulty rank estimation to their kata, but they do not decide on its final rank. Difficulty rank is estimated from votes cast by users who solved the kata while it's in beta phase. This means that every user who solved a beta kata can give their own opinion on its difficulty, helping in determining its final rank.
+
+### Reviewing Code
+
+Users with sufficient priviledges can enter kata editor and view the code of all code snippets. Reading through them is a great way to find potential issues with tests, examples, or reference solution. Reviewers can check insides of the kata and verify whether it stands up to [Kata authoring best practices](/recipes/create-kata/best-practices/). Any discovered problems can be fixed right away in the code, or reviewer can raise issues or suggest improvements in kata discourse.
```

Comment:
- "enter kata editor" --> "enter the kata editor"
- "or reviewer can ..." --> "or the reviewer can ..." / "or reviewers can ..."
- "in kata discourse" --> "in the kata discourse"

Reply:
```suggestion
Users with sufficient privileges can enter the kata editor and view the code of all code snippets. Reading through them is a great way to find potential issues with tests, examples, or reference solution. Reviewers can check insides of the kata and verify whether it stands up to [Kata authoring best practices](/recipes/create-kata/best-practices/). Any discovered problems can be fixed right away in the code, or reviewers can raise issues or suggest improvements in the kata discourse.
```